understanding/20/link-purpose-in-context.html

```diff
@@ -1,403 +1,187 @@
+         purpose of the link is.</p>
       <p>The Success Criterion includes an exception for links for which the purpose of the
          link cannot be determined from the information on the Web page. In this situation,
          the person with the disability is not at a disadvantage; there is no additional context
          available to understand the link purpose. However, whatever amount of context is available
          on the Web page that can be used to interpret the purpose of the link must be made
          available in the link text or programmatically associated with the link to satisfy
-         the Success Criterion.
-      </p>
-      
+         the Success Criterion.</p>
       <div class="note">
-         
          <p>There may be situations where the purpose of the link is is supposed to be unknown
             or obscured. For instance, a game may have links identified only as door #1, door
             #2, and door #3. This link text would be sufficient because the purpose of the links
-            is to create suspense for all users.
-         </p>
-         
+            is to create suspense for all users.</p>
       </div>
-      
-      <p>See also 
-         <a href="link-purpose-link-only">2.4.9: Link Purpose (Link Only)</a>.
-      </p>
-      
-      
+      <p>As <q>link text together with its <a>programmatically determined link context</a></q> may
+         only benefit assistive technology users, see also <a href="link-purpose-link-only">2.4.9 Link Purpose (Link Only)</a>,
+         which only relies on the link text itself.</p>
    </section>
    <section id="benefits">
       <h2>Benefits of Link Purpose (In Context)</h2>
-      
-      
       <ul>
-         
-         <li>This Success Criterion helps people with motion impairment by letting them skip links
-            that they are not interested in, avoiding the keystrokes needed to visit the referenced
-            content and then returning to the current content.
-         </li>
-         
-         <li>People with cognitive limitations will not become disoriented by multiple means of
-            navigation to and from content they are not interested in.
-         </li>
-         
-         <li>
-            People with visual disabilities will be able to determine the purpose of a link by
-            exploring the link's context. 
-            
-            
-         </li>
-         
+         <li>Links whose link text clearly conveys their purpose can help people who navigate with a
+            keyboard or keyboard-like interface, letting them skip links that they are not interested in,
+            avoiding the effort to visit the referenced content only to find it is not what they wanted, and
+            having to then navigate back.</li>
+         <li>When the link text is clear, people with cognitive limitations are less likely to become disoriented
```

Comment:
link text, or a link's purpose (again, re: link text _and_ surrounding text are sufficient for this particular SC)

Reply:
surrounding text is not the only programmatically determinable context, which complicates matters. heck, adjacent text (in a separate paragraph, and even when not associated programmatically, can help COGA users who don't rely on AT...that's the complication/weirdness of this SC)

Reply:
yah again, get that.  i'm fine with resolving this particular comment since i have the other one open still.